packages/packages/top-bar/src/components/locations/utilities-menu-location.tsx

```diff
@@ -1,14 +1,16 @@
 import { Fragment } from 'react';
 import { useInjectionsOf } from '@elementor/locations';
 import ToolbarMenu from '../ui/toolbar-menu';
-import { LOCATION_UTILITIES_MENU_DEFAULT } from '../../locations';
+import { getMenuId } from '../../locations';
 import { Divider } from '@elementor/ui';
 import ToolbarMenuMore from '../ui/toolbar-menu-more';
 
 const MAX_TOOLBAR_ACTIONS = 3;
 
 export default function UtilitiesMenuLocation() {
-	const injections = useInjectionsOf( LOCATION_UTILITIES_MENU_DEFAULT );
+	const injections = useInjectionsOf(
+		getMenuId( 'utilities', 'default' )
```

Comment:
```suggestion
		getMenuId( 'utilities' )
```

Make the 'default' default

Reply:
Done